update-all-pipeline-images.sh

```diff
@@ -17,9 +17,12 @@ fi
 for instance in $(calyptia get core_instances -o json | jq -cr '.[].id')
 do
     echo "Updating for instance: $instance"
-    for pipeline in $(calyptia get pipelines --core-instance "$instance" -o json | jq -cr '.[].id')
+    for pipeline in $(calyptia get pipelines --core-instance "$instance" -o json | jq -cr '.[].name')
```

Comment:
I think I used ID to ensure we did not ever have a conflict with the same names - the update command does not take the instance so I don't think we should do this. 
It's perfectly reasonable and even expected for people to reuse pipeline names I think.